src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -393,7 +393,7 @@ public void controlAllowedPUT() {
                 + " WHERE { }";
 
         // Verify that non-admin user can update ACL
-        doPatch(aclUri, new Headers(new Header("Content-Type", APPLICATION_SPARQL_UPDATE)), body, false);
+        doPutUnverified(aclUri, new Headers(new Header("Content-Type", APPLICATION_SPARQL_UPDATE)), body, false);
```

Comment:
What does PUT with `Content-Type` of "application/sparql-update" mean? When I try to perform such a PUT on an "acl" or container, I get an error.

Reply:
Oops